### PR TITLE
Support non-identifier visibility like pub(crate)

### PR DESCRIPTION
**Pull request by djc**
This also upgrades to syn-1.

Sorry again for the callout on Twitter, I normally don't worry much about "public" Twitter mentions (and DMs don't always work) but should have appreciated that others might.

### CODE REVIEW

**Comment by est31** [comment]
This would be very helpful for nushell (https://github.com/nushell/nushell/issues/362), maybe @djc even made it with nushell in mind. After this a new crates.io release would be really great. Thanks!

**Comment by djc** [comment]
This was indeed done with nu shell in mind. 👍

File: Cargo.toml [hunk 1]
```diff
@@ -22,6 +22,6 @@ travis-ci = { repository = "Hoverbear/getset" }
 proc-macro = true
 
 [dependencies]
-quote = "0.6.10"
-syn = "0.15.21"
-proc-macro2 = { version = "0.4.24", default-features = false }
+quote = "1"
```
Comment (Hoverbear) [inline Cargo.toml]: Thanks!!!!

**Comment by Hoverbear** [comment]
Thanks for this @djc , been on vacation this last week. :)

**Comment by SteveLauC** [comment]
It would be great if this is documented in the get-started example:)

**Pull request merged**